printer_mods/Majarspeed/README.md

```diff
@@ -1 +1,6 @@
 Simple mount for Hartk board for galileo.
```

Comment:
This is the wrong README for all this text.  All this text should go into `printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md`

Reply:
I have it in both of them.